collectors/okta/health_checks.js

```diff
@@ -0,0 +1,41 @@
+/* -----------------------------------------------------------------------------
+ * @copyright (C) 2019, Alert Logic, Inc
+ * @doc
+ *
+ * Okta health checks.
+ *
+ * @end
+ * -----------------------------------------------------------------------------
+ */
+
+ 'use strict';
+
+ const https = require('https');
+ const PawsCollector = require('@alertlogic/paws-collector').PawsCollector;
+ 
+    function oktaTokenHealthCheck(asyncCallback){
+        /* No explicit call to validate an API token, the token expires after 30 days, this 30 day timeout is refreshed 
+         * upon any API calls. This call will refresh token, without being time consuming/destructive
+         */
+        var orgUrl = process.env.paws_endpoint;
+        var token = String.concat("SSWS ", PawsCollector._pawsCreds.secret);
```

Comment:
this is fine, but as a poin you can just concat strings in JS with a `+` operator.

`"SSWS " + PawsCollector._pawsCreds.secret`